File: app/src/main/java/com/originb/inkwisenote2/modules/noterelation/data/TextProcessingJobStatus.java
package com.originb.inkwisenote2.modules.noterelation.data;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.io.Serializable;

@Getter
@Setter
@AllArgsConstructor
public class TextProcessingJobStatus implements Serializable {
    private Long noteId;
    private String stage; // values are from TextProcessingStage
}

File: app/src/main/java/com/originb/inkwisenote2/modules/noterelation/data/TextProcessingJobStatus.kt
package com.originb.inkwisenote2.modules.noterelation.data

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.Setter
import java.io.Serializable

@Getter
@Setter
@AllArgsConstructor
class TextProcessingJobStatus : Serializable {
    private val noteId: Long? = null
    private val stage: String? = null // values are from TextProcessingStage
}
